schemas/HPXMLBaseElements.xsd

```diff
@@ -1831,6 +1832,7 @@
 										</xs:annotation>
 									</xs:element>
 									<xs:element minOccurs="0" name="HasMixingValve" type="HPXMLBoolean"/>
+									<xs:element minOccurs="0" name="MixingValueSetpoint" type="TemperatureGreaterThanZero"/>
```

Comment:
What about changing this to

```suggestion
									<xs:element minOccurs="0" name="MixingValueOutletTemperature" type="TemperatureGreaterThanZero"/>
```

to be consistent with `HotWaterTemperature`?

Reply:
Maybe nitpicky, but: it won't always maintain the setpoint temperature in practice. If for whatever reason the tank outlet temp is colder than the mixing valve setpoint temp, it won't really be the outlet temperature. I like setpoint slightly better for that reason, but it is definitely nitpicky.

Reply:
We'll discuss this w/ Noel when he gets back and then this should be good to merge.